### PR TITLE
Kjroelke/issue-8

### LINKED ISSUE #8: Update API and other fixes
-   [ ] Add ability to inject classes
- [ ] Update base markup
- [ ] Add SEO Box
- [ ] Swap lite-vimeo packages for CNO Authored one

### PR DESCRIPTION
**Pull request by kjroelke**
fixes #8

### CODE REVIEW

File: template-parts/content-featured-news.php [hunk 1]
```diff
@@ -74,7 +74,10 @@
 			while ( $featured_posts_query->have_posts() ) {
 				$featured_posts_query->the_post();
 				echo '<li class="col-12 mb-4">';
-				// get_template_part( 'template-parts/content', 'news-preview' );
+				$template_override = get_template_part( 'template-parts/content', 'featured-news' );
```
Comment (blake-perkins-dev) [inline template-parts/content-featured-news.php]: get_template_part is targeting the file this code is in. Is this intentional?

File: template-parts/content-featured-news.php [hunk 2]
```diff
@@ -74,7 +74,10 @@
...
+				if ( false === $template_override ) {
+					require dirname( __DIR__ ) . '/template-parts/content-featured-news.php';
```
Comment (blake-perkins-dev) [inline template-parts/content-featured-news.php]: Also targeting the file this code is in.

**Review by blake-perkins-dev — CHANGES_REQUESTED** [review]
@kjroelke - Just need to confirm that those 2 spots are are referencing the right file. It appears that it's checking to see if itself exists and then requiring it.

File: template-parts/content-featured-news.php (hunk 1 above)
Comment (kjroelke) [reply template-parts/content-featured-news.php]: @blake-perkins-dev that's a good catch

**Review by blake-perkins-dev — APPROVED** [review]

**Pull request merged**